<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>them_dai_ly</class>
 <widget class="QDialog" name="them_dai_ly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm đại lý</string>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 22pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Hồ sơ đại lý</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Ten_dl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_email_dl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_luu_dl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(121, 240, 223);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sdt_dl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_phone">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Số điện thoại</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_fullname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tên</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_diachi">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Địa chỉ:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_huy_dl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(121, 240, 223);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Hủy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_diachi_dl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_email">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
